tests/page/page-emulate-media.spec.ts

```diff
@@ -64,7 +64,7 @@ it('should throw in case of bad colorScheme argument', async ({ page }) => {
   let error = null;
   // @ts-expect-error 'bad' is not a valid media type
   await page.emulateMedia({ colorScheme: 'bad' }).catch(e => error = e);
-  expect(error.message).toContain('colorScheme: expected one of (dark|light|no-preference|null)');
+  expect(error.message).toContain('colorScheme: expected one of (dark|light|no-preference|no-override)');
```

Comment:
This is actually wrong now, because user never passes `no-override`, they pass `null`.

Reply:
Yeah, but they also never see this message because of the type checks.